src/components/MailboxThread.vue

```diff
@@ -477,15 +488,23 @@ export default {
 }
 .list__wrapper {
 	display: flex;
-	flex: 1 0 0;
+	flex: 1 1 auto;
 	flex-direction: column;
 	height: 100%;
+	overflow: hidden;
 }
 :deep(.app-details-toggle) {
 	opacity: 1;
 }
  // temporary fix
 :deep(.app-content-list) {
 	min-height: 0;
+	position: absolute;
+	overflow: scroll;
+	width: 100% !important;
+	top: 52px;
+}
+:deep(.app-content-wrapper.app-content-wrapper--no-split.app-content-wrapper--show-details) {
```

Comment:
This will break as soon as the component is changed upstream.

However, I didn't dig into the problem and have no other solution.

Reply:
we run in the same problem over and over again. If i dont add this, we will have 2 scroll bars. I dont have any other idea either without breaking something else.